linfa-svm/README.md

```diff
@@ -0,0 +1,24 @@
+# Support Vector Machines
```

Comment:
Fantastic addition - thank you for writing this up!

Reply:
Great to hear :)